### PR TITLE
Moved Meetup #4 to past meetup and created placeholder for Meetup #5

### PR DESCRIPTION
**Pull request by WendellAdriel**

### CODE REVIEW

File: _meetups/meetup-5.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: 'Meetup #5'
+excerpt: 'More info to be added soon!'
+date: 'TBD'
+coverImage: '/assets/images/php-lisbon.png'
+ogImage:
+url: '/assets/images/php-lisbon.png'
```
Comment (AndreVarandas) [inline _meetups/meetup-5.md:7]: You just missed one little space here 🚀

Comment (WendellAdriel) [reply _meetups/meetup-5.md]: Oh, that's, true, thanks!

Comment (WendellAdriel) [reply _meetups/meetup-5.md]: Fixed it, @AndreVarandas 😉

Comment (AndreVarandas) [reply _meetups/meetup-5.md]: Thanks you so much @WendellAdriel 🚀

Comment (WendellAdriel) [reply _meetups/meetup-5.md]: I'm happy to help, @AndreVarandas!!! 💪🏼

**Pull request merged**